internal/stream_fetch_helper.go

```diff
@@ -18,19 +18,29 @@ type LogFetchSettings interface {
 	GetEndTsEnv() string
 	GetDstEnv() string
 	GetLogFolderPath() string
-	GetFilePath(string, string) (string, error)
+}
+
+type LogFetchParams interface {
+	GetStorageFolder() storage.Folder
+	GetStartTs() time.Time
+}
```

Comment:
https://ulearn.me/Course/cs2/Service_Locator_44348db6-ec79-4012-a5aa-f46159d69ce8

Reply:
I don't see a problem to pass it like params.